arbitrum-docs/node-running/how-tos/build-nitro-locally.md

```diff
@@ -110,7 +110,7 @@ sudo mkdir -p /usr/local/bin
 sudo ln -s  /opt/homebrew/opt/llvm/bin/wasm-ld /usr/local/bin/wasm-ld
 ```
 
-### Step 5. Configure Node [16.19](https://github.com/nvm-sh/nvm)
+### Step 5. Configure node [16.19](https://github.com/nvm-sh/nvm)
```

Comment:
This one should be uppercase I think, because it refers to NodeJS

Reply:
Oh you're right, nvm then!